terraform/layer1-aws/aws-eks.tf

```diff
@@ -72,60 +63,10 @@ module "eks" {
 
   vpc_id = module.vpc.vpc_id
 
-  cluster_addons = local.eks_addons
-
-  cluster_encryption_config = var.eks_cluster_encryption_config_enable ? [
```

Comment:
Why has this configuration been deleted?

Reply:
For what kind of resources?

Reply:
for secrets

Reply:
https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/variables.tf#L105
https://github.com/terraform-aws-modules/terraform-aws-eks/blob/master/main.tf#L61